frontend/src/components/Search/Search.vue

```diff
@@ -160,6 +160,48 @@
 								@blur="lostFocusOnSearch"
 							/>
 							<AccountLink :address="contract.creator.asString" />
+							<div>
+								Age
+								<Tooltip
+									:text="formatTimestamp(contract.blockSlotTime)"
+									:position="index === 0 ? tooltipPositionBottom : ''"
+								>
+									{{
+										convertTimestampToRelative(
+											contract.blockSlotTime || '',
```

Comment:
I would assume that it is not possible to convert the empty string into a relative timestamp, as the empty string does not parse into a valid date. So we need to handle that better.

Reply:
Removed the empty string - copy pasted from elsewhere but doesn’t make sense. Thanks :-)